src/main/resources/logback.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this whole file should be deleted from the PR

Reply:
fine with this